tt_metal/fabric/mesh_graph_descriptor.cpp

```diff
@@ -738,7 +738,7 @@ GlobalNodeId MeshGraphDescriptor::populate_mesh_instance(
 }
 
 GlobalNodeId MeshGraphDescriptor::populate_device_instance(LocalNodeId local_id, std::vector<GlobalNodeId>& hierarchy) {
-    const std::string name = "D" + std::to_string(local_id);
+    const std::string name = std::string("D") + std::to_string(local_id);
```

Comment:
Note the (since C++26): https://en.cppreference.com/w/cpp/string/basic_string/to_string

Based on that, here's my suggestion:
```cpp
InstanceData data{
    .local_id = local_id,
    .name = std::format("D{}", local_id),
    ...
```

Reply:
Also anywhere I put `std::format`, feel free to replace with `fmt::format`, should be interchangeable. I don't want to open up that particular can of worms atm though, so just do what you feel is appropriate.